axiom/tests/test_distributed_trace.c

```diff
@@ -1351,6 +1352,19 @@ static void test_create_trace_state_header(void) {
                         nr_strstr(result, expected));
   nr_free(result);
 
+  /*
+   * Test: locale is set to use `,` instead of `.` for decimal values
+   */
+  setlocale(LC_NUMERIC, "pl_PL");
```

Comment:
This seems a good place to verify the locate switch was successful or fail this test case.

Reply:
attempted to add this test here: 535f58950e96d177603bb38511a78b5c283e8e3c
but it appears to be causing test failures on the PR runner despite passing successfully on my local machine: https://github.com/newrelic/newrelic-php-agent/actions/runs/7835406513/job/21380743765?pr=828

possibly a result of the images the PR runner depends on not having locales installed? 
Error message is a very vague `SIGNAL 11`

Reply:
Changed test logic to call setlocale directly in the tlib test function and got a more meaningful output here: https://github.com/newrelic/newrelic-php-agent/actions/runs/7835578673/job/21381288423?pr=828#step:9:46
It looks like the builder images are incapable of changing the locale.

Reply:
For now I think we'll have to remove this test to allow the PR runner to pass while we bake locales into the builder images

Reply:
removed for now in 8668b955262e9463f790ebe58e2c005648636881